scripts/deployment/deploy.sh

```diff
@@ -361,7 +361,9 @@ deploy_openlibrary() {
 # - deploy.sh olsystem
 # - deploy.sh openlibrary
 
-if [ "$1" == "olsystem" ]; then
+if [ "$1" == "crons" ]; then
+    check_crons
+elif [ "$1" == "olsystem" ]; then
```

Comment:
I had added it intentionally, I thought it was useful to have a quick way to check if crons are running
but c'est la vie

```suggestion
if [ "$1" == "olsystem" ]; then
```